src/_utils/item/Item.tsx

```diff
@@ -18,6 +20,8 @@ export interface ItemProps {
   readonly highlighted?: boolean
   readonly isClickable?: boolean
   readonly leftTitle?: string
+  readonly isLeftTitleInfo?: boolean
+  readonly leftTitleInfoIcon?: string
```

Comment:
leftTitleInfoIcon >> leftTitleIconInfo

Reply:
That's a lot of props you add here. Maybe we can go down to 2 (naming proposal, do what you want with this)
- infoIconLabel
- infoIconCallback 
We won't (i hope) have these icons elsewhere, do the `left` part may be ... left aside.

The `isLeftTitleInfo` seems to be unnecesserrary: you can check either the label or the function

Reply:
Finally removed